README.md

```diff
@@ -34,16 +34,14 @@ Python package for interacting with CloudOS
       - [Import a Nextflow Workflow](#import-a-nextflow-workflow)
     - [Nextflow Jobs](#nextflow-jobs)
       - [Submit a Job](#submit-a-job)
-      - [Get Job Logs](#get-job-logs)
-      - [Get Job Results](#get-job-results)
-      - [Clone or Resume job](#clone-or-resume-job)
-      - [Query working directory of job](#query-working-directory-of-job)
-      - [Abort single or multiple jobs from CloudOS](#abort-single-or-multiple-jobs-from-cloudos)
-      - [Abort Jobs](#abort-jobs)
       - [Check Job Status](#check-job-status)
+      - [List Jobs](#list-jobs)
+      - [Get Job Results](#get-job-results)
       - [Get Job Details](#get-job-details)
       - [Get Job Workdir](#get-job-workdir)
-      - [List Jobs](#list-jobs)
+      - [Get Job Logs](#get-job-logs)
+      - [Clone or Resume Job](#clone-or-resume-job)
+      - [Abort Jobs](#abort-jobs)
```

Comment:
the order is still incorrect. In reality, it is:
- get job results
- clone or resume job
- abort jobs
- get job details
- get job workdir
- (job logs is missing)